modules/ROOT/pages/planning-and-tuning/operators/index.adoc

```diff
@@ -767,6 +767,13 @@ It uses a bidirectional breadth-first search (BFS) algorithm, which performs two
 |
 |
 
+| xref::planning-and-tuning/operators/operators-detail.adoc#query-plan-subtraction-node-by-labels-scan[SubtractionNodeByLabelsScan]
```

Comment:
Should not Partitioned Subtraction Node By Labels Scan also be added?

Reply:
yes, I'll fix that

Reply:
Fixed now